cypress/integration/lpa/cases/search.spec.js

```diff
@@ -0,0 +1,33 @@
+describe("Searching for a case", { tags: ["@lpa", "@smoke-journey"] }, () => {
+  beforeEach(() => {
+    cy.loginAs("LPA Manager");
+    cy.createDonor().then(({ id: donorId }) => {
+      cy.createLpa(donorId).then(({ id: lpaId }) => {
+        cy.wrap(donorId).as("donorId");
+        cy.wrap(lpaId).as("lpaId");
+      });
+    });
+  });
+
+  it("selected search result navigates user to the correct case", function () {
+    cy.visit(`/lpa/home`);
+
+    cy.get("#searchKeyword").type('Bob Sponge');
+    cy.get(".search-form > form").submit();
+
+    cy.get("ul[role=list]").first().contains('Bob Sponge');
+    cy.get("ul[role=list]").last().contains('Bob Sponge');
+
+    cy.get("[role=list] > :nth-child(4)").contains("Case:")
+      .then($searchResult => {
+        const lpaId = $searchResult.text().replace(/[^\d-]/g, '');
+        cy.wrap(lpaId).as("lpaId");
+        cy.wrap($searchResult).click();
+      });
+
+    cy.get("@lpaId").then(lpaId => {
+      cy.contains(`${this.lpaId}`);
+      cy.contains('Bob Sponge');
+    });
```

Comment:
"4" is a magic number here and won't continue to work if other tests also add donors called "Bob Sponge". Instead, it might be preferable to find the link with the UID (which you'd need to retrieve around line 5):
```suggestion
    cy.get("[role=list]").contains(this.lpaUid).click();

    cy.get("h1").contains("Bob Sponge");
```

Reply:
I've changed the test to enter the UID as the search text, because that LPA wasn't appearing in the first 10 results